cura/Machines/MachineErrorChecker.py

```diff
@@ -212,4 +212,4 @@ def _setResult(self, result: bool, keys_to_recheck = None) -> None:
         self._check_in_progress = False
         self.needToWaitForResultChanged.emit()
         self.errorCheckFinished.emit()
-        Logger.log("i", "Error check finished, result = %s, time = %0.1fs", result, time.time() - self._start_time)
+        Logger.log("i", "Error check finished, result = %s, time = %0.1fs", result, time.time() - self._check_start_time)
```

Comment:
Use f-string and `info` method
```suggestion
        execution_time = time.time() - self._check_start_time
        Logger.info(f"Error check finished, result = {result}s, time = {execution_time:.2f}s")
```